book/book.toml

```diff
@@ -0,0 +1,10 @@
+[book]
+authors = ["Chinedu Francis Nwafili"]
+multilingual = false
+src = "src"
+title = "The Mdbook Superimport Book"
+
+# @superimport start super-section
+[preprocessor.superimport]
```

Comment:
This is how preprocessors get added, right?

Reply:
Yup!